index.js

```diff
@@ -289,6 +287,12 @@ function getAdditionalMiddleware() {
   }
 }
 
+function getMethodDoc(methodHandler) {
+  return Array.isArray(methodHandler) ?
+    methodHandler.slice(-1)[0].apiDoc || methodHandler.apiDoc :
```

Comment:
Yes, it's better if you don't mind BC.

The order becomes problem only when:
```js
module.exports.post = [function (res, req, next) { next();}, post];
function post(res, req, next) { res.status(200}.json({message:'ok'});}
post.apiDoc = {
  // api doc 1
};
module.exports.post.apiDoc = {
  // api doc 2
};
```

But, this case seems very rare. So, no need to mind BC.